new: usr: TARGET_DATE defaults to yesterday.

ashmastaflash opened the pull request:
Also supports HALO_API_HOSTNAME

Closes #4

mong2 commented on README.md:
```diff
@@ -40,7 +40,7 @@ S3 bucket you need to drop the events into.
 * HALO_API_SECRET_KEY
 * HALO_API_HOSTNAME (optional, defaults to `api.cloudpassage.com`)
-* TARGET_DATE: Formatted like this: "2016-12-01"
+* TARGET_DATE: Formatted like this: "2016-12-01" (optional, defaults to yesterday)
```
should specify using iso 8601 format

ashmastaflash replied:
Fixed.

eatwithforks commented on tool/eventslib/utility.py:
```diff
@@ -1,8 +1,24 @@
 
 class Utility(object):
+    @classmethod
+    def iso8601_yesterday(cls):
```
can pull this function out and just use iso8601_arbitrary_days_ago and pass in 1. 
kind of redundant to use iso8601_yesterday.

ashmastaflash replied:
done

ashmastaflash committed:
chg: usr: Target date defaults to yesterday.

If no TARGET_DATE is specified, the prior day is set.

Supports HALO_API_HOSTNAME environment variable.

Closes #4

ashmastaflash committed:
Merge branch 'develop' into feature/CS-555
